### YamlMime:PythonClass
uid: azure.servicebus.exceptions.ServiceBusQuotaExceededError
name: ServiceBusQuotaExceededError
fullName: azure.servicebus.exceptions.ServiceBusQuotaExceededError
module: azure.servicebus.exceptions
inheritances:
- azure.servicebus.exceptions.ServiceBusError
summary: 'The quota applied to a Service Bus resource has been exceeded while interacting
  with the Azure Service Bus service.


  The messaging entity has reached its maximum allowable size, or the maximum number
  of connections to a namespace

  has been exceeded. Create space in the entity by receiving messages from the entity
  or its subqueues.'
constructor:
  syntax: ServiceBusQuotaExceededError(**kwargs)
methods:
- uid: azure.servicebus.exceptions.ServiceBusQuotaExceededError.add_note
  name: add_note
  summary: "Exception.add_note(note) \u2013\nadd a note to the exception"
  signature: add_note()
- uid: azure.servicebus.exceptions.ServiceBusQuotaExceededError.raise_with_traceback
  name: raise_with_traceback
  signature: raise_with_traceback()
- uid: azure.servicebus.exceptions.ServiceBusQuotaExceededError.with_traceback
  name: with_traceback
  summary: "Exception.with_traceback(tb) \u2013\nset self.__traceback__ to tb and\
    \ return self."
  signature: with_traceback()
attributes:
- uid: azure.servicebus.exceptions.ServiceBusQuotaExceededError.args
  name: args
